README.md

```diff
@@ -28,7 +28,20 @@ To see Blazor in action, check out [Steve Sanderson's prototype demo at NDC Oslo
 
 ## Getting Started
 
-We are still *very early* in this project. There isn't yet anything you can download nor any project template you can use. Most of the planned features aren't implemented yet. Even the parts that are already implemented aren't yet optimized for minimal payload size. If you're keen, you can clone the repo, build it, and run the samples and tests.
+1. Install the [.NET Core 2.1 Preview 1 SDK](https://www.microsoft.com/net/download/dotnet-core/sdk-2.1.300-preview1).
```

Comment:
Should we also mention that you must *not* also have Preview 2 installed? If you do, then it will get used by default and give build warnings, and publish will fail.

We could add a `global.json` to the template to avoid this if we wanted.

Reply:
> We could add a `global.json` to the template to avoid this if we wanted.

Yeah, that sounds like something we should do

Reply:
#300

Reply:
The template install using `dotnet new` isn't working. Is it something isn't released yet or is it wrong (or am I in error?)
```
C:\Users\Howard>dotnet new -i Microsoft.AspNetCore.Blazor.Templates --nuget-source https://dotnet.myget.org/F/blazor-dev/api/v3/index.json
  Restoring packages for C:\Users\Howard\.templateengine\dotnetcli\v2.1.300-preview1-008174\scratch\restore.csproj...
C:\Users\Howard\.templateengine\dotnetcli\v2.1.300-preview1-008174\scratch\restore.csproj : error NU1103: Unable to find a stable package Microsoft.AspNetCore.Blazor.Templates with version
C:\Users\Howard\.templateengine\dotnetcli\v2.1.300-preview1-008174\scratch\restore.csproj : error NU1103:   - Found 46 version(s) in Blazor [ Nearest version: 0.0.5-preview1-10033 ]
C:\Users\Howard\.templateengine\dotnetcli\v2.1.300-preview1-008174\scratch\restore.csproj : error NU1103:   - Found 0 version(s) in C:\Program Files\dotnet\sdk\NuGetFallbackFolder
  Generating MSBuild file C:\Users\Howard\.templateengine\dotnetcli\v2.1.300-preview1-008174\scratch\obj\restore.csproj.nuget.g.props.
  Generating MSBuild file C:\Users\Howard\.templateengine\dotnetcli\v2.1.300-preview1-008174\scratch\obj\restore.csproj.nuget.g.targets.
  Restore failed in 1.73 sec for C:\Users\Howard\.templateengine\dotnetcli\v2.1.300-preview1-008174\scratch\restore.csproj.
```

Reply:
Right now you need to specify a package version: `Microsoft.AspNetCore.Blazor.Templates::0.1.0-*`.

Once we build the actual 0.1.0 version of the package this should no longer be necessary, as the latest stable package version will get used by default.